Tests/Common/IsolatorLimitResultProviderTests.cs

```diff
@@ -75,10 +74,12 @@ public void ConsumeRequestsAdditionalTimeAfterOneMinute()
             Assert.AreEqual(0, provider.Invocations.Count);
 
             timeProvider.Advance(TimeSpan.FromSeconds(15));
-            Thread.Sleep(15);
+
+            while (provider.Invocations.Count == 0)
+                Thread.Sleep(15);
```

Comment:
Cool 👍 ! 
Shouldn't wait for ever though, should fail after X count, to avoid hanging the test on failure.
nit - Leans code style always uses `{ }`

Reply:
Sounds good